pkg/ride/tree_estimatorV4.go

```diff
@@ -59,6 +60,7 @@ func (e *treeEstimatorV4) estimate() (int, int, map[string]int, error) {
 	}
 	vc := 0
 	if e.tree.HasVerifier() {
+		e.scope.resetFunctions()
```

Comment:
Why we should reset functions? I assume that verifier creates new scope. Is it correct?

Reply:
Same for verfiier, it's just a last callable if present.